build/NuGet.ruleset

```diff
@@ -10,6 +10,9 @@
   <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp.CodeStyle" RuleNamespace="Microsoft.CodeQuality.Analyzers">
     <Rule Id="IDE0055" Action="Warning" />
   </Rules>
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp.NetAnalyzers" RuleNamespace="Microsoft.CodeAnalysis.CSharp.NetAnalyzers">
+    <Rule Id="CA1309" Action="Error" />
+  </Rules>
```

Comment:
Not related to the PR, but rulesets are considered deprecated. Consider moving to globalconfigs or editorconfigs